ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/EditOwner.chtml.cs

```diff
@@ -57,35 +57,27 @@ public async Task<ActionResult> OnGet()
 			return Page();
 		}
 
-		public async Task<ActionResult> OnPost(string selectedOwner,string currentOwner,bool valuePicked,int valueInList)
+		//public async Task<ActionResult> OnPost(string selectedOwner,string currentOwner,bool valuePicked,int valueInList)
```

Comment:
QUESTION:

Do we need the commented code?

Reply:
Nope, we absolutely don't : ) Code REmoved.